README.md

```diff
@@ -1,16 +1,130 @@
-# Title
+# Exploring Git Repositories
 
 ## Learning Goals
 
--SWBAT 1
--SWBAT 2
+- Review a repository's history with `git log`
+- Focus on certain changes with `git diff`
+- Unstage changes with `git reset`
+- Revert changes with `git checkout`
 
 ## Introduction
 
-## SWBAT 1
+Git has important major features like branching and merging, but there are also
+several Git commands developers use frequently for smaller workflow tasks.
+Learning how to integrate them into your own workflow will give you some extra
+speed and efficiency.
 
-## SWBAT 2
+## Review a Repository's History with `git log`
+
+As we add and commit changes, we are creating a Git log that we can refer back
+to when needed. Run the command:
+
+```bash
+$ git log
+```
+
+and we'll see an output of all the commits made in that repository.
+
+Retrieving a Git log is especially useful when we've cloned someone else's
+repository and want to review the changes made to it. We can retrace their steps
+and build out our understanding of why the code evolved the way it did. We can
+do the same thing even if the repository is our own. Sometimes it feels like our
+past coding self is a different person and we need to re-evaluate our old code
+based on new things we've learned since. `git log` helps us do this.
+
+> **Tip** By adding certain flags to the `git log` command, we can see just the
+> history of commits made over the last few commits, within time frames or by a
+> particular author. Check out the documentation to see all the ways to customize
+> your commands.
```

Comment:
Would this be a good place to talk about the flag that allows us to see which files were changed in a commit? I find that one useful.